maro/communication/proxy.py

```diff
@@ -220,7 +276,7 @@ def receive(self, is_continuous: bool = True):
         """
         return self._driver.receive(is_continuous)
 
-    def receive_by_id(self, session_id_list: list) -> List[Message]:
+    def receive_by_id(self, session_ids: Union[list, str, SendAgain]) -> Union[List[Message], SendAgain]:
         """Receive target messages from communication driver.
 
```

Comment:
PendingToSend instance, which includes ori session id, ori message id.
Use exception, and try-catch.